src/plugins/timelist/CompactViewItem.vue

```diff
@@ -0,0 +1,134 @@
+ Administration. All rights reserved.
+
+ Open MCT is licensed under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0.
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ License for the specific language governing permissions and limitations
+ under the License.
+
+ Open MCT includes source code licensed under additional open source
+ licenses. See the Open Source Licenses file (LICENSES.md) included with
+ this source code distribution or the Licensing information page available
+ at runtime from the About dialog for additional information.
+-->
+<template>
+  <div :class="listItemClass">
+    <div class="c-tli__activity-color">
+      <div class="c-tli__activity-color-swatch" :style="styleClass"></div>
+    </div>
+    <div class="c-tli__title-and-bounds">
+      <div class="c-tli__title">{{ formattedItemValue.title }}</div>
+      <div class="c-tli__bounds">
+        <span class="c-tli__duration">{{ formattedItemValue.duration }}</span>
+        <span class="c-tli__start-time">{{ formattedItemValue.start }}</span>
+        <span class="c-tli__end-time">{{ formattedItemValue.end }}</span>
+      </div>
+    </div>
+    <div class="c-tli__progress-pie">
+      <svg viewBox="0 0 100 100">
+        <circle class="c-svg-progress__bg" r="50" cx="50" cy="50"></circle>
+        <path id="svg-progress-path" class="c-svg-progress__progress"></path>
+        <circle
+          class="c-svg-progress__ticks"
+          r="40"
+          cx="50"
+          cy="50"
+          stroke-dasharray="3 7.472"
+        ></circle>
+      </svg>
+    </div>
+    <div class="c-tli__time-hero">
+      <div class="c-tli__time-hero-context">{{ formattedItemValue.label }}</div>
+      <div class="c-tli__time-hero-time --is-countdown">{{ formattedItemValue.countdown }}</div>
+    </div>
+  </div>
+</template>
+
+<script>
+const CURRENT_CSS_SUFFIX = '--is-current';
+const PAST_CSS_SUFFIX = '--is-past';
+const FUTURE_CSS_SUFFIX = '--is-future';
+
+const ITEM_COLORS = {
+  [CURRENT_CSS_SUFFIX]: '#ffcc00',
+  [PAST_CSS_SUFFIX]: '#0088ff',
+  [FUTURE_CSS_SUFFIX]: '#7300ff'
+};
+
+const ITEM_STATES = {
+  notStarted: 'not-started',
+  inProgress: 'in-progress',
+  completed: 'completed',
+  aborted: 'aborted',
+  skipped: 'skipped',
+  incomplete: 'incomplete',
+  overdue: 'overdue',
+  runningLong: 'running-long'
+};
+
+export default {
+  inject: ['openmct'],
+  props: {
+    item: {
+      type: Object,
+      required: true
+    },
+    itemProperties: {
+      type: Array,
+      required: true
+    }
+  },
+  data() {
+    return {
+      itemState: ITEM_STATES.notStarted
+    };
+  },
+  computed: {
```

Comment:
Yes agreed, this is going to change after the activity states PR is merged.

Reply:
I don't understand. This PR depends on #7370. When that PR is merged, this logic is going to change. It was stubbed in so that Charles and I could work simultaneously on this PR.

Reply:
Ah, gotcha. If I understand correctly you're saying we'll fix this in #7370?